src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java

```diff
@@ -136,6 +171,63 @@ public void identify(IdentifyPayload identify) {
     }
   }
 
+  private void incrementAndSetTraits(Map<String, Object> traits) {
+    Identify identify = new Identify();
```

Comment:
Instead of the two for loops, this can be written as:

```
for trait in traits {
  if traitsToIncrement contains trait {
    incrementTrait()
  } else {
    setTrait()
  }
}

```

This also avoids having to create a copy the traits passed in by the user.